pkg/ccl/backupccl/backup_test.go

```diff
@@ -6816,10 +6818,12 @@ func TestPaginatedBackupTenant(t *testing.T) {
 			for _, ru := range request.Requests {
 				if exportRequest, ok := ru.GetInner().(*roachpb.ExportRequest); ok &&
 					!isLeasingExportRequest(exportRequest) {
-					exportRequestSpans = append(
-						exportRequestSpans,
-						requestSpanStr(roachpb.Span{Key: exportRequest.Key, EndKey: exportRequest.EndKey}, exportRequest.ResumeKeyTS),
-					)
+					req := requestSpanStr(roachpb.Span{Key: exportRequest.Key, EndKey: exportRequest.EndKey}, exportRequest.ResumeKeyTS)
+					if _, found := exportRequestSpansSet[req]; found {
+						return nil // nothing to do
+					}
+					exportRequestSpansSet[req] = struct{}{}
```

Comment:
Doesn't this want a mutex? Even if we "know" that there are no concurrent SSTs in this test, it would be better not to put in this assumption.

Reply:
Done.